Makefile

```
@@ -177,186 +184,214 @@ function cleanup { \
+	$(IMAGE_BUILDER) buildx use project-v3-builder
+	- $(IMAGE_BUILDER) buildx build --push --platform=$(PLATFORMS) --tag $(OPERATOR_IMG) -f Dockerfile.cross .
+	- $(IMAGE_BUILDER) buildx rm project-v3-builder
+else ifeq ($(IMAGE_BUILDER), podman)
+	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build -f Dockerfile.cross --manifest $(OPERATOR_IMG) --platform $(PLATFORMS) . ; \
+	if [ "${MANIFEST_PUSH}" = "true" ] ; then \
+		$(IMAGE_BUILDER) manifest push $(OPERATOR_IMG) $(OPERATOR_IMG) ; \
+	fi
+else
+	$(error unsupported IMAGE_BUILDER: $(IMAGE_BUILDER))
 endif
+	rm Dockerfile.cross
 
-# Undeploy controller from the configured Kubernetes cluster in ~/.kube/config
-.PHONY: undeploy
-undeploy:
-	- $(CLUSTER_CLIENT) delete --ignore-not-found=$(ignore-not-found) -f config/samples/operator_v1beta1_cryostat.yaml
-	- $(CLUSTER_CLIENT) delete --ignore-not-found=$(ignore-not-found) -f config/samples/operator_v1beta1_clustercryostat.yaml
-	- $(KUSTOMIZE) build config/default | $(CLUSTER_CLIENT) delete --ignore-not-found=$(ignore-not-found) -f -
+.PHONY: catalog-build
+catalog-build: opm ## Build for catalog.
+	$(OPM) index add --container-tool $(IMAGE_BUILDER) --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+
+.PHONY: bundle
+bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
+	$(OPERATOR_SDK) generate kustomize manifests -q
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMG)
+	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
+	$(OPERATOR_SDK) bundle validate ./bundle
+
+.PHONY: bundle-build
+bundle-build: ## Build the bundle image.
+	$(IMAGE_BUILDER) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+
+##@ Development
 
-# Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
-manifests: controller-gen
+manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) rbac:roleName=role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 	envsubst < hack/image_tag_patch.yaml.in > config/default/image_tag_patch.yaml
 	envsubst < hack/image_pull_patch.yaml.in > config/default/image_pull_patch.yaml
 
-# Run go fmt against code
 .PHONY: fmt
-fmt: add-license
+fmt: add-license ## Run go fmt against code.
 	go fmt ./...
 
-# Run go vet against code
 .PHONY: vet
-vet:
+vet: ## Run go vet against code.
 	go vet ./...
 
-# Generate code
 .PHONY: generate
-generate: controller-gen
+generate: controller-gen ## Generate code.
 	go generate ./...
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
-# Check and add (if missing) license header
 LICENSE_FILE = $(shell pwd)/LICENSE
 GO_PACKAGES := $(shell go list -test -f '{{.Dir}}' ./... | sed -e "s|^$$(pwd)||" | cut -d/ -f2 | sort -u)
 .PHONY: check-license
-check-license: golicense
+check-license: golicense ## Check and add (if missing) license header.
```

```suggestion
check-license: golicense ## Check if license header is missing from any code files.
```

This just check right?